Index: app/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from fastapi import FastAPI\r\n\r\napp = FastAPI()\r\n\r\n@app.get(\"/\")\r\ndef read_root():\r\n    return {\"message\": \"Hello World\"}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/main.py b/app/main.py
--- a/app/main.py	(revision 3afae65b310e0e388c1e859b5427ab1886dd6e09)
+++ b/app/main.py	(date 1777495179833)
@@ -1,7 +1,6 @@
 from fastapi import FastAPI
+from app.routers import users
 
 app = FastAPI()
 
-@app.get("/")
-def read_root():
-    return {"message": "Hello World"}
\ No newline at end of file
+app.include_router(users.router)
\ No newline at end of file
